gr-qtgui/include/gnuradio/qtgui/TimeRasterDisplayPlot.h

```diff
@@ -40,8 +40,17 @@ class TimeRasterDisplayPlot : public DisplayPlot
                    setColorMapTitleFontSize)
 
 public:
-    TimeRasterDisplayPlot(
-        int nplots, double samp_rate, double rows, double cols, QWidget*);
+    TimeRasterDisplayPlot(int nplots,
```

Comment:
These signatures get a little wild. Is there a way to use X & Y structs? Does that translate to Python well?

Reply:
What about set_x_axis(), set_x_label(), etc., similar to the way time_sink and others work?